jungle/ec2.py

```diff
@@ -116,17 +116,21 @@ def create_ssh_command(instance_id, instance_name, username, key_file, port, ssh
             target_instances = []
             for idx, i in enumerate(instances):
```

Comment:
I'm quite newbie at Python, and I'm not sure about this, but this loop seems to be rewrite as:
```python
  target_instances = list(instances)
```